test/scale-infra/deploy_ks_core.yaml

```diff
@@ -0,0 +1,104 @@
+---
+- hosts: masters
+  remote_user: ubuntu
+  become: yes
+  gather_facts: no
+  vars:
+    ks_release:  "{{ ks_release }}"
+  tasks:
+    - name: install jq and make
+      shell: |
+             sudo apt update
+             sudo apt install jq make -y
+
+    - name: install golang
+      shell: |
+             curl -LO https://go.dev/dl/go1.23.5.linux-amd64.tar.gz
+             rm -rf /usr/local/go && tar -C /usr/local -xzf go1.23.5.linux-amd64.tar.gz
+             export PATH=$PATH:/usr/local/go/bin
+             go version
+                  
+    - name: install yq
+      shell: |
+             sudo wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64
+             sudo chmod a+x /usr/local/bin/yq
+
+    - name: install OCM CLI
+      shell: curl -L https://raw.githubusercontent.com/open-cluster-management-io/clusteradm/main/install.sh | bash
+        
+    - name: install KubeFlex
+      shell: |
+             sudo su <<EOF
+             bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubeflex/main/scripts/install-kubeflex.sh) --ensure-folder /usr/local/bin --strip-bin
+             EOF
+
+    - name: install Helm
+      shell: |
+             curl https://baltocdn.com/helm/signing.asc | gpg --dearmor | sudo tee /usr/share/keyrings/helm.gpg > /dev/null
+             sudo apt-get install apt-transport-https --yes
+             echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/helm.gpg] https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
+             sudo apt-get update
+             sudo apt-get install helm
+
+    - name: Register a variable, ignore errors and continue
```

Comment:
This task name is pretty uninformative. What variable? What value? What purpose?

Reply:
I revised the name of this task with this following one: 
``
        - name: Check if kscore context exists, ignore errors and continue
``